bluebinder.c

```diff
@@ -283,16 +394,12 @@ setup_watch(
 {
     proxy->channel = g_io_channel_unix_new(proxy->host_fd);
 
-    if (!proxy->channel) {
-        return false;
-    }
-
     g_io_channel_set_encoding(proxy->channel, NULL, NULL);
     g_io_channel_set_buffered(proxy->channel, FALSE);
 
     proxy->gio_channel_event_id = g_io_add_watch_full(
         proxy->channel,
-        G_PRIORITY_DEFAULT_IDLE + 2, // see host_write_packet
+        G_PRIORITY_DEFAULT + 2, // see host_write_packet
```

Comment:
Better add a comment that `+2` actually means lower priority. And better yet, define all these priorities somewhere at the top of the file, close to each other, with a short explanation why those are different.